pg-index-health/src/main/java/io/github/mfvanek/pg/checks/cluster/NotValidConstraintsOnCluster.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2019-2024. Ivan Vakhrushev and others.
+ * https://github.com/mfvanek/pg-index-health
+ *
+ * This file is a part of "pg-index-health" - a Java library for
+ * analyzing and maintaining indexes health in PostgreSQL databases.
+ *
+ * Licensed under the Apache License 2.0
+ */
+
+package io.github.mfvanek.pg.checks.cluster;
+
+import io.github.mfvanek.pg.checks.host.NotValidConstraintsOnHost;
+import io.github.mfvanek.pg.connection.HighAvailabilityPgConnection;
+import io.github.mfvanek.pg.model.constraint.Constraint;
+
+import javax.annotation.Nonnull;
+
+public class NotValidConstraintsOnCluster extends AbstractCheckOnCluster<Constraint> {
```

Comment:
Please rename to `NotValidConstraintsCheckOnCluster`.   
See https://github.com/mfvanek/pg-index-health/pull/374#discussion_r1581845740